e2e-tests-rust/tests/lib.rs

```diff
@@ -8,19 +8,21 @@ use alloy::{
 };
 use anvil_zksync_e2e_tests::{
     init_testing_provider, init_testing_provider_with_client, AnvilZKsyncApi, ReceiptExt,
-    ZksyncWalletProviderExt, DEFAULT_TX_VALUE,
+    ZksyncWalletProviderExt, DEFAULT_TX_VALUE, get_node_binary_path
 };
 use http::header::{
     HeaderMap, HeaderValue, ACCESS_CONTROL_ALLOW_HEADERS, ACCESS_CONTROL_ALLOW_METHODS,
     ACCESS_CONTROL_ALLOW_ORIGIN, ORIGIN,
 };
-use std::convert::identity;
-use std::time::Duration;
 
 const SOME_ORIGIN: HeaderValue = HeaderValue::from_static("http://some.origin");
 const OTHER_ORIGIN: HeaderValue = HeaderValue::from_static("http://other.origin");
 const ANY_ORIGIN: HeaderValue = HeaderValue::from_static("*");
 
+use anvil_zksync_core::node::VersionedState;
+use std::{ fs, convert::identity, thread::sleep, time::Duration };
+use tempdir::TempDir;
```

Comment:
Please move these above the constants. Also seems like rustfmt+clippy are not enforced here so I created a separate issue https://github.com/matter-labs/anvil-zksync/issues/505

In general I recommend enabling rustfmt-based auto-formatting in your editor (not sure what you use but there should be an option for almost anything)

Reply:
Opened PR for #505 here: https://github.com/matter-labs/anvil-zksync/pull/506